keps/sig-auth/20190730-oidc-discovery.md

```diff
@@ -138,16 +175,31 @@ In addition, the API server will serve a JWKS consisting of the public keys from
 public keys used for valid KSA tokens. The OIDC discovery document will point to
 the JWKS path as the `jwks_uri`.
 
+Finally, it will be possible for users to override the `jwks_uri` via a new
+`--service-account-jwks-uri` flag. We do attempt to construct a default JWKS URI
```

Comment:
If I understand this correctly, this is purely to support having a publicly exposed JWKS that is different from the API server external address?  IIRC GKE does this today?

Reply:
Correct.